sdk/spring/spring-cloud-azure-feature-management/src/main/java/com/azure/spring/cloud/feature/management/implementation/models/RecurrenceRange.java

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.spring.cloud.feature.management.implementation.models;
+
+import com.azure.spring.cloud.feature.management.implementation.timewindow.TimeWindowUtils;
+import com.azure.spring.cloud.feature.management.implementation.timewindow.recurrence.RecurrenceConstants;
+
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+
+/**
+ * The recurrence range specifying how long the recurrence pattern repeats
+ * */
+public class RecurrenceRange {
+    /**
+     * The recurrence range type. Default value is "NoEnd"
+     * */
+    private RecurrenceRangeType type = RecurrenceRangeType.NOEND;
+
+    /**
+     * The date to stop applying the recurrence pattern
+     * */
+    private ZonedDateTime endDate = Instant.ofEpochMilli(Integer.MAX_VALUE).atZone(ZoneOffset.UTC);
```

Comment:
What is the intent of this? Is it just an unrealistic end date to cause errors? Also, why not just 0 instead of max int? The number of milliseconds of a max int is just a few days. So instead of 01-01-1970 it's 01-25-1970?

Reply:
Removed this default value.